tests/admin_views/urls.py

```diff
@@ -12,5 +12,7 @@
     url(r'^test_admin/admin4/', customadmin.simple_site.urls),
     url(r'^test_admin/admin5/', admin.site2.urls),
     url(r'^test_admin/admin7/', admin.site7.urls),
+    url(r'^test_admin/admin8/', (admin.site.get_urls(), 'admin', 'admin8'),
+        {"extra_context": dict(foo="bar")}),
```

Comment:
Might as well use single quotes here to match the surrounding style.

Also, would this work with `{'extra_context': dict()}`? I don't believe you need to actually pass any values into the context here to test what you want to test.